data/remnant ships.txt

```diff
@@ -131,32 +135,168 @@ ship "Starling" "Starling (Thrasher)"
+		"engine capacity" 91
+		"shield generation" 2.7
+		"shield energy" 1.8
+		"hull repair rate" 1
+		"hull energy" 0.75
+		"cloak" .02
+		"cloaking energy" 5
+		"cloaking fuel" .2
+		"outfit scan power" 12
+		"outfit scan speed" 1
+		"tactical scan power" 26
+		weapon
+			"blast radius" 80
+			"shield damage" 800
+			"hull damage" 400
+			"hit force" 1200
+	outfits
+		"Inhibitor Cannon" 1
+		"Point Defense Turret"
+
+		"Epoch Cell"
+		"Crystal Capacitor"
+		"Emergency Ramscoop" 2
+		"Quantum Key Stone"
+		"Tuning Rifle" 3
+
+		"Forge-Class Thruster"
+		"Forge-Class Steering"
+		"Hyperdrive"
+
+	engine -12 70 0.7
+	engine 12 70 0.7
+	engine 0 78 1
+	gun 0 -96 "Inhibitor Cannon"
+	gun -7 -94
+	gun 7 -94
+	turret 0 26 "Point Defense Turret"
+	explode "tiny explosion" 40
+	explode "small explosion" 20
+	explode "medium explosion" 10
+	"final explode" "final explosion small" 1
+	description "While the Starling was the first successful ship to use technology derived from alien ruins, the Gull is the most recent. The current increase in Korath activities has caused foraging for resources to become a risky endeavor, forcing the Remnant to develop a specialized freighter for hauling cargo from remote sites without revealing their presence to hostile aliens."
+	description "	The Gull has quickly proven its worth, allowing the Starling fleets to return to riskier activities while the Gull takes over the hazardous convoy routes. It is now the primary element of the Remnant's extra-territorial resource operations."
+	description "	The cloaking technology is built into the Gull's hull; it cannot be removed or transferred to another ship."
+
+ship "Gull" "Gull (Heavy)"
+	outfits
+		"Thrasher Cannon" 3
+		"Thrasher Turret"
+
+		"Epoch Cell"
+		"Millennium Cell"
+		"Crystal Capacitor"
+		"Tuning Rifle" 4
+
+		"Forge-Class Thruster"
+		"Forge-Class Steering"
+		"Hyperdrive"
+
+ship "Gull" "Gull (Sniper)"
+	outfits
+		"Inhibitor Cannon" 3
+		"Thrasher Turret"
+
+		"Epoch Cell"
+		"Millennium Cell"
+		"Crystal Capacitor"
+		"Tuning Rifle" 4
+
+		"Forge-Class Thruster"
+		"Forge-Class Steering"
+		"Hyperdrive"
+
+ship "Gull" "Gull (Support)"
+	outfits
+		"EMP Torpedo Bay"
+		"Inhibitor Cannon" 2
+		"Point Defense Turret"
+		"EMP Torpedo" 5
+
+		"Epoch Cell"
+		"Millennium Cell"
+		"Crystal Capacitor"
+		"Tuning Rifle" 3
+
+		"Forge-Class Thruster"
+		"Forge-Class Steering"
+		"Hyperdrive"
+
+ship "Gull" "Gull (Cargo)"
+	outfits
+		"Cargo Expansion" 4
+
+		"Epoch Cell"
+		"Crystal Capacitor"
+		"Tuning Rifle" 3
+
+		"Forge-Class Thruster"
+		"Forge-Class Steering"
+		"Hyperdrive"
+
+ship "Gull" "Gull (Bunks)"
+	outfits
+		"Bunk Room" 4
+
+		"Epoch Cell"
+		"Crystal Capacitor"
+		"Tuning Rifle" 3
+
+		"Forge-Class Thruster"
+		"Forge-Class Steering"
+		"Hyperdrive"
+
+
+
 ship "Albatross"
 	plural "Albatrosses"
 	sprite "ship/albatross"
 	thumbnail "thumbnail/albatross"
 	attributes
 		category "Heavy Warship"
 		licenses
-			Remnant
+			"Remnant Capital License"
```

Comment:
Simply put "Remnant Capital", as this currently makes the ship require a "Remnant Capital License License".